ls1intum/Artemis
# Fix undefined array access on exam checklist with empty exercise groups

### Checklist

### Motivation and Context


When there is a exercise group in an exam with no exercises, the exam checklist component tries to count the exercise by explicitly calling the 0 index on it's array, this fails. 

### Description

This adds an additional check.

### Steps for Testing


1. Create an exam with at least one exercise group but no exercises
2. open the exam checklist and observe the developer console. There should be no errors.

### Test Coverage



Unchanged.
